internal/operator/cluster/standby.go

```diff
@@ -213,8 +213,8 @@ func EnableStandby(clientset kubernetes.Interface, cluster crv1.Pgcluster) error
 	// Delete the "leader" configMap
 	if err = clientset.CoreV1().ConfigMaps(namespace).Delete(ctx, leaderConfigMapName, metav1.DeleteOptions{}); err != nil &&
 		!kerrors.IsNotFound(err) {
-		log.Error("Unable to delete configMap %s while enabling standby mode for cluster "+
-			"%s: %v", leaderConfigMapName, clusterName, err)
+		log.Error("Unable to delete configMap "+leaderConfigMapName+" while enabling standby mode for cluster "+
+			leaderConfigMapName+":"+clusterName, err)
```

Comment:
🔧 Better to change `log.Error` to `log.Errorf`, I think?

Reply:
Agreed. This fix actually makes more sense elsewhere -- moving it to [PR 3597](https://github.com/CrunchyData/postgres-operator/pull/3597).